src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -475,12 +484,19 @@ public String getQuestionResultStatisticsCsv(
      * From the feedback responses, generate a mapping of the option to a list of points received for that option.
      * The key of the map returned is the option name / recipient's participant identifier.
      * The values of the map are list of points received by the key.
+     * The map is sorted in ascending order.
      * @param responses  a list of responses
      */
     private Map<String, List<Integer>> generateOptionPointsMapping(
             List<FeedbackResponseAttributes> responses) {
         
-        Map<String, List<Integer>> optionPoints = new HashMap<String, List<Integer>>();
+        Map<String, List<Integer>> optionPoints = new TreeMap<String, List<Integer>>(new Comparator<String>() {
+            @Override
+            public int compare(String s1, String s2) {
+                return s2.compareTo(s1);
+            }
+        });
```

Comment:
By previous implementation do you mean performing the options sort for the html and csv statistics separately, within their respective methods? Yup, it should be alright to do so as long as you are able to sort the options correctly 👍